### ISSUE TITLE
Need some help with manipulating the data

### ISSUE THREAD

**Issue by julianheavyside** [julianheavyside/diversification#19]
@mwpennell still having trouble wrapping my head around subsetting the data for plotting. I keep going back and forth between trying to do it with dplyr, and trying to do it only with ggplot. I have a simple function to filter the data according to a set of conditions (which will define facet windows in plotting). Is this okay? It's not standardized, but it works for now.
https://github.com/julianheavyside/diversification/blob/master/tidying-functions.R#L10-L21

I can easily filter out three tree sizes (e.g. 100, 400, 700) for three facet columns. I'm struggling with the different combos of rate parameters. The data allow for many different combinations, ranging from 0.1- 1.0. Here are some possibilities:
     - equal q01 and q10, all values under 0.5
     - equal q01 and q10, all values over 0.5
     - unequal rates, with q01 > q10, and difference between q01 and q10 > 0.5 (or something)
     - unequal rates, with q01 < q10, and difference between q01 and q10 > 0.5

I'm having a hard time finding ggplot tips applicable to the faceting I want to do. I'm trying with facet_grid(). 
https://github.com/julianheavyside/diversification/blob/master/analysis.R#L76

Some other problems/questions:
     - When I colour the points according to "bias", a continuum of one colour is used, suggesting that ggplot thinks bias is continuous, instead of categorical with several levels. For some reason I'm unable to figure out how to coerce bias to categorical. 

     - Am I trying to do too much with dplyr before plotting? Can I do more filtering within ggplot?

**Comment by julianheavyside** [comment]
@mwpennell Maybe I'll just do these combos for faceting rate parameters. I know I should just write a function to set these, but I'm having a hard time thinking about it to begin with.

1. q01=q10=0.2

2. q01=q10=0.5

3. q01=q10=0.8

4. q01=0.2, q10=0.8

5. q01=0.8, q10=0.2

**Comment by mwpennell** [comment]
Yes, this seems like a reasonable approach to the problem. For the purposes of plotting you may want to:
1. use dplyr to filter the data down to a couple of parameter combinations for rates and tree sizes (like you don't need the range right now, just small, medium, large) 

2. facet the resulting data.frame with ggplot so that the facets are `rates~tree_size`

does this make sense? are you still having trouble with the faceting issue or is this now solved?
